### PR TITLE
Commits for custom website

### PR DESCRIPTION
**Pull request by arijit1111**
Add API details 
Import Custom website
Update stock from custom website
Fetch custom website information

### CODE REVIEW

File: Modules/Custom/Routes/api.php [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+| is assigned the "api" middleware group. Enjoy building your API!
+|
+*/
+
+Route::middleware('auth:api')
+    ->prefix('custom')
+    ->name('custom.')
+    ->group(function () {
+        Route::post('/add-api', 'CustomController@addApi')->name('addApi');
+        Route::post('/import-product', 'CustomController@importProduct')->name('importProduct');
```
Comment (smaqeelkazmi) [inline Modules/Custom/Routes/api.php:21]: @arijit1111 The route `import-product` and `add-api` have same business logic, why we created two separate?

**Review by smaqeelkazmi — CHANGES_REQUESTED** [review]
I have made a comment for duplicate route, kindly check. After that we are good to merge @arijit1111

File: Modules/Custom/Routes/api.php (hunk 1 above)
Comment (arijit1111) [reply Modules/Custom/Routes/api.php:21]: No this not same routes.
For import-product routes to upload product excelsheet from custom webste.
For add-api routes to add api information of custom website like API URL, API KEY and API PASSWORD

**Review by smaqeelkazmi — APPROVED** [review]
PR approved

**Pull request merged**